korangar/src/graphics/passes/point_shadow/shader/entity.wgsl

```diff
@@ -62,12 +71,20 @@ fn vs_main(
     output.curvature = vertex.curvature_multiplier;
     output.original_depth_offset = instance.depth_offset;
     output.original_curvature = instance.curvature;
+    output.angle = instance.angle;
     return output;
 }
 
 @fragment
 fn fs_main(input: VertexOutput) -> @builtin(frag_depth) f32 {
-    let diffuse_color = textureSample(texture, nearest_sampler, input.texture_coordinates);
+    var new_input = input.texture_coordinates;
```

Comment:
Dito as in the bindless point shader file. I won't repeat this for the other shaders.

Reply:
Changed the information from every shader.